vllm/v1/worker/gpu/model_runner.py

```
@@ -1201,7 +1239,128 @@ def _zero_cudagraph_capture_kv_blocks(self) -> None:
+                    request.tracker_key,
+                    request.ownership_step,
+                    request.ownership_start_tokens,
+                    request.ownership_scheduled_tokens,
+                )
+            )
+
+        def resolve(
+            _num_sampled_tokens: np.ndarray,
+            num_rejected_tokens: np.ndarray,
+        ) -> None:
+            for snapshot, num_rejected in zip(snapshots, num_rejected_tokens):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)